driver-core/src/main/java/com/datastax/driver/core/querybuilder/BuiltStatement.java

```diff
@@ -77,7 +77,7 @@
 
     private final List<ColumnMetadata> partitionKey;
     private final List<Object> routingKeyValues;
-    final String keyspace;
+    String keyspace;
```

Comment:
I wonder if we should consider differentiating between keyspace provided with `BuildStatement` (i.e. what comes from `insertInto(keyspace, table)` and otherwise in some way, as it is redundant to send the keyspace with the statement if it's already set in the query.

Reply:
Had separate discussion that we were ok with this.